src/client/components/RetrivingCvs/RetrivingCvs.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import { CardCvViewSearchComponent } from '../CardCvViewSearchComponent/CardCvViewSearchComponent';
+
+export default function RetriveCvs() {
+  const [cvs, setCvs] = useState([]);
+  useEffect(() => {
+    (async () => {
+      const result = await fetch('/api/usercv').then((res) => res.json());
+      console.log(result);
+      setCvs(result);
+    })();
+  }, []);
+  console.log(cvs);
+  return (
+    <div>
+      {cvs.slice(-3).map((item) => (
```

Comment:
Why is this sliced?

Reply:
to upload only 3 last uploaded cvs,. The original array will not be modified, but the task requires that we show only 3 latest cvs,.,Thats why i use slice like in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice ,.,.,under parameters

Reply:
you can use conditions instead by using array length and boolean value